config/main.py

```diff
@@ -838,6 +858,9 @@ def shutdown(ctx):
     config_db = ctx.obj['config_db']
     interface_name = ctx.obj['interface_name']
 
+    if interface_name_is_valid(interface_name) is False:
+        ctx.fail("Enter valid interface name!!")
```

Comment:
Same as above.

Reply:
Hi Joe, can you please complete the review?

Reply:
Joe, all the suggested changes are addressed.